packages/lesswrong/server/resolvers/ultraFeedResolver.ts

```diff
@@ -182,11 +199,23 @@ const createSourcesMap = (
     }
   });
 
-  if (sources.recentComments) {
-    commentThreadsItems.forEach(t => {
-      sources.recentComments?.items.push({ type: "feedCommentThread", feedCommentThread: t });
-    });
-  }
+  subscribedPostItems.forEach(p => {
+    const postId = p.postId;
+    if (!postId || addedPostIds.has(postId)) {
+      return;
+    }
+    const bucket = sources['subscriptions' as FeedItemSourceType]; 
```

Comment:
It'd be good to figure out a workflow where I don't need to manually ask for type casts to be removed (or else in this case ask why we're performing a type cast that actually needs to be there)